lib/model/model.go

```diff
@@ -2891,3 +2935,14 @@ func writeEncryptionToken(token []byte, cfg config.FolderConfiguration) error {
 		Token:    token,
 	})
 }
+
+func fatalErr(err error) error {
```

Comment:
This look like a `util.NewFatal(error, status) error` kind of thing

Reply:
(I also hate the util package and think we should split it into svcutil and probably configutil or something... for another day.)